pkg/router/router.go

```diff
@@ -218,6 +218,8 @@ func registerEndpoints(r endpoints.Resource, container *restful.Container) {
 	wsv1.Route(wsv1.POST("/{namespace}/rerun").To(r.RerunPipelineRun))
 	wsv1.Route(wsv1.GET("/{namespace}/ingress").To(r.GetIngress))
 	wsv1.Route(wsv1.GET("/{namespace}/endpoints").To(r.GetEndpoints))
+	wsv1.Route(wsv1.GET("/{namespace}/dashboardversion").To(r.GetDashboardVersion))
+	wsv1.Route(wsv1.GET("/{namespace}/pipelineversion").To(r.GetPipelineVersion))
```

Comment:
Do we need to include the namespace in the URL?

Reply:
removed these now in favour of using the properties